pom.xml

```diff
@@ -11,7 +11,7 @@
   <properties>
     <enforcer-plugin.version>3.1.0</enforcer-plugin.version>
     <maven-minimum-version>3.8.6</maven-minimum-version>
-    <java-version>11</java-version>
+    <java-version>17</java-version>
```

Comment:
There are two other references to Java 11 below (lines 491 and 492). I don't know if they need to be updated.

Reply:
Updated to use `${java.version}` and forcing a run of the sonar-scan.